korangar/src/graphics/passes/forward/shader/entity_bindless.wgsl

```diff
@@ -180,7 +203,7 @@ fn fs_main(input: VertexOutput) -> FragmentOutput {
     }
 
     var output: FragmentOutput;
-    output.fragment_color = vec4<f32>(final_color, 1.0);
+    output.fragment_color = vec4<f32>(final_color, final_alpha);
```

Comment:
You set the real alpha here. Should we also only disgard the fragment on depth == 0.0 ?

Reply:
It the final alpha for transparency blending later.